apache/incubator-metron
# METRON-338 Add indexing rpm. Remove dead flux code from metron-parsers

Addresses https://issues.apache.org/jira/browse/METRON-338

**To test:**

There are a few things to check

1. You can check that the parser rpm does not include the flux files any longer - https://github.com/apache/incubator-metron/pull/191#issue-165652191
2. Also check that an indexing rpm is created
3. Spin up a vagrant instance and verify parser -> enrichment -> indexing still works as intended.
